test/integration/parachain/release/redeem.test.ts

```diff
@@ -86,56 +121,93 @@ describe("redeem", () => {
             await userInterBtcAPI.redeem.burn(amountToSteal, collateralCurrency);
-        });
-    }).timeout(18 * 60000);
+        }
+    }).timeout(10 * 60000);
 
     // TODO: Unskip after `subscribeToRedeemExpiry` is reimplemented
+    // TODO: rewrite test to ban one vault, and check the other currency still works
     it.skip("should cancel a redeem request", async () => {
-        await runWhileMiningBTCBlocks(bitcoinCoreClient, async () => {
-            const issueAmount = newMonetaryAmount(0.00005, wrappedCurrency, true);
-            const redeemAmount = newMonetaryAmount(0.00003, wrappedCurrency, true);
-            const initialRedeemPeriod = await userInterBtcAPI.redeem.getRedeemPeriod();
-            await sudo(userInterBtcAPI, () => userInterBtcAPI.redeem.setRedeemPeriod(1));
-            const [, redeemRequest] = await issueAndRedeem(userInterBtcAPI, userBitcoinCoreClient, userAccount, vaultToBanId, issueAmount, redeemAmount, false, ExecuteRedeem.False);
-            // Wait for redeem expiry callback
-            await new Promise<void>((resolve, _) => {
-                // redeemAPI.subscribeToRedeemExpiry(newAccountId(api, userAccount.address), (requestId) => {
-                //     if (stripHexPrefix(redeemRequest.id.toString()) === stripHexPrefix(requestId.toString())) {
-                //         resolve();
-                //     }
-                // });
+        for (const vaultToBanId of vaultToBanIds) {
+            await runWhileMiningBTCBlocks(bitcoinCoreClient, async () => {
+                const issueAmount = newMonetaryAmount(0.00005, wrappedCurrency, true);
+                const redeemAmount = newMonetaryAmount(0.00003, wrappedCurrency, true);
+                const initialRedeemPeriod = await userInterBtcAPI.redeem.getRedeemPeriod();
+                await sudo(userInterBtcAPI, () => userInterBtcAPI.redeem.setRedeemPeriod(1));
+                const [, redeemRequest] = 
+                    await issueAndRedeem(
+                        userInterBtcAPI, 
+                        userBitcoinCoreClient, 
+                        userAccount, 
+                        vaultToBanId, 
+                        issueAmount, 
+                        redeemAmount, 
+                        false, 
+                        ExecuteRedeem.False
+                    );
+                // TODO: in the promise: 
+                // grab head
+                //  wait for head + redeem period + 1
+                // then resolve
+                // Wait for redeem expiry callback
+                await new Promise<void>((resolve, _) => {
+                    // redeemAPI.subscribeToRedeemExpiry(newAccountId(api, userAccount.address), (requestId) => {
+                    //     if (stripHexPrefix(redeemRequest.id.toString()) === stripHexPrefix(requestId.toString())) {
+                    //         resolve();
+                    //     }
+                    // });
+                });
```

Comment:
We had a chat last week with @nud3l to pare down the scope of tests to add/fix to get multi-collateral vaults unblocked sooner rather than later. 
This test fell victim to that.